File: examples/integrity/send/send.go
package main

import (
	"log"
	"os"

	bloock "github.com/bloock/bloock-sdk-go/v2"
	"github.com/bloock/bloock-sdk-go/v2/client"
	"github.com/bloock/bloock-sdk-go/v2/entity/record"
)

func main() {
	// we set the API key and create a client
	bloock.ApiKey = os.Getenv("API_KEY")
	recordClient := client.NewRecordClient()
	integrityClient := client.NewIntegrityClient()

	// we create an array of strings which will contain
	// the hashes of the records we want to send
	r, err := recordClient.FromString("Hello world").Build()
	if err != nil {
		log.Println(err)
	}
	records := []record.Record{r}

	// finally we can send the records
	receipt, err := integrityClient.SendRecords(records)
	if err != nil {
		log.Println(err)
	}

	// we get a receipt with informationa about the transaction
	log.Printf("%+v", receipt)
}

File: examples/Send.java
import com.bloock.sdk.Bloock;
import com.bloock.sdk.client.IntegrityClient;
import com.bloock.sdk.client.RecordClient;
import com.bloock.sdk.entity.integrity.RecordReceipt;
import com.bloock.sdk.entity.record.Record;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class Send {
  public static void main(String[] args) throws Exception {
    try {
      // we set the API key and create a client
      Bloock.apiKey = System.getenv("API_KEY");
      RecordClient recordClient = new RecordClient();
      IntegrityClient integrityClient = new IntegrityClient();

      // we create an ArrayList of strings which will contain
      // the hashes of the records we want to send
      Record record = recordClient.fromString("Hello world").build();
      ArrayList<Record> records = new ArrayList<>(Collections.singletonList(record));

      // finally we can send the records
      List<RecordReceipt> receipts = integrityClient.sendRecords(records);

      // we get a receipt with informationa about the transaction
      System.out.println(receipts);
    } catch (Exception e) {
      System.out.println(e);
    }
  }
}